tests/UI/campaigns/functional/BO/13_shopParameters/05_contact/stores/04_sortStores.js

```diff
@@ -0,0 +1,180 @@
+
+// Import test context
+const testContext = require('@utils/testContext');
+
+const baseContext = 'functional_BO_shopParameters_contact_store_sortStores';
+
+// Import expect from chai
+const {expect} = require('chai');
+
+
+// Browser and tab
+let browserContext;
+let page;
+
+
+let numberOfStores = 0;
+
+describe('Sort stores by id, name, address, city, postal code, state and country', async () => {
+  // before and after functions
+  before(async function () {
+    browserContext = await helper.createBrowserContext(this.browser);
+    page = await helper.newTab(browserContext);
+  });
+
+  after(async () => {
+    await helper.closeBrowserContext(browserContext);
+  });
+
+  it('should login in BO', async function () {
+    await loginCommon.loginBO(this, page);
+  });
+
+  it('should go to contact page', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'goToContactPage', baseContext);
+
+    await dashboardPage.goToSubMenu(
+      page,
+      dashboardPage.shopParametersParentLink,
+      dashboardPage.contactLink,
+    );
+
+    await contactPage.closeSfToolBar(page);
+
+    const pageTitle = await contactPage.getPageTitle(page);
+    await expect(pageTitle).to.contains(contactPage.pageTitle);
+  });
+
+  it('should go to stores page', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'goToStoresPage', baseContext);
+
+    await contactPage.goToStoresPage(page);
+
+    const pageTitle = await storesPage.getPageTitle(page);
+    await expect(pageTitle).to.contains(storesPage.pageTitle);
+  });
+
+  it('should reset all filters and get number of stores in BO', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'resetFilterFirst', baseContext);
+
+    numberOfStores = await storesPage.resetAndGetNumberOfLines(page);
+    await expect(numberOfStores).to.be.above(0);
+  });
+
+
+  describe('Sort stores', async () => {
+    const sortTests = [
+      {
+        args: {
+          testIdentifier: 'sortByIdDesc', sortBy: 'id_store', sortDirection: 'down', isFloat: true,
+        },
+      },
+      {
+        args: {
+          testIdentifier: 'sortByNameAsc', sortBy: 'sl!name', sortDirection: 'up',
+        },
+      },
+      {
+        args: {
+          testIdentifier: 'sortByNameDesc', sortBy: 'sl!name', sortDirection: 'down',
+        },
+      },
+      {
+        args: {
+          testIdentifier: 'sortByAddressAsc', sortBy: 'sl!address1', sortDirection: 'up',
+        },
+      },
+      {
+        args: {
+          testIdentifier: 'sortByAddressDesc', sortBy: 'sl!address1', sortDirection: 'down',
+        },
+      },
+      {
+        args: {
+          testIdentifier: 'sortByCityAsc', sortBy: 'city', sortDirection: 'up',
+        },
+      },
+      {
+        args: {
+          testIdentifier: 'sortByCityDesc', sortBy: 'city', sortDirection: 'down',
+        },
+      },
+
+      /* Sort by post code not working, skipping it
```

Comment:
We need to keep track of this somewhere.

Reply:
I will mention this in the issue that i just created (after merging the PR). I don't know if it's enough. WDYT ?

Reply:
I guess.